dev-docs/modules/consentManagement.md

```diff
@@ -25,8 +25,8 @@ Designed to support the EU General Data Protection Regulation ([GDPR](https://ww
 
 This module will perform its tasks with the CMP prior to the auction starting.  A rough synopsis of this interaction process would be:
 
-1. Fetch the user's encoded consent string from the CMP.
-2. With a valid consent string, we will incorporate this data into the auction objects (for adapters to collect) and then allow the auction to proceed.
+1. Fetch the user's encoded consent string from the CMP (note - in the the case of a new user, the CMP will respond only once there is consent information available; ie they picked their consent choices).
```

Comment:
Should this also say that the module will also time out and continue|abort the request depending on which happens first?